blog/_posts/2022-08-08-managing-tunnels-with-argocd.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: post
+title: Managing Inlets tunnels with Argo CD
```

Comment:
We have quite a few how-to posts, so how about: "How To Manage Inlets Tunnels With ArgoCD and GitOps"